### PR TITLE
Add help command

### PR DESCRIPTION
**Pull request by chrismclennon**
This adds in a help command requested in Issue #62

### CODE REVIEW

**Comment by joaorafaelm** [comment]
@chrismclennon Hi Chris, thank you very much for the pull request, could you add the tests for the help command? it probably will be very similar to this one 
https://github.com/joaorafaelm/markov-bot/blob/master/test/test_markov.py#L143

File: test/test_markov.py [hunk 1]
```diff
@@ -156,3 +156,9 @@ def test_notify_admin(mock_settings, mock_bot):
         mock_settings.ADMIN_CHAT_ID,
         message
     )
+
+
+@mock.patch('markov.bot')
+def test_help(mock_bot, message):
+    markov.help(message)
+    assert mock_bot.reply_to.mock_model.called_once_with(message)
```
Comment (joaorafaelm) [inline test/test_markov.py]: i think just `assert mock_bot.reply_to.called is True` is enough

Comment (chrismclennon) [reply test/test_markov.py]: Ah yup that makes sense, I agree.

**Comment by julianolf** [comment]
Hey @chrismclennon, can I make one last suggestion?

Instead of use **MarkovBot** in the message I think it would look nicer to use the bot's instance username.

You can get the username this way `{bot.get_me().username}`.

**Comment by chrismclennon** [comment]
Sure I can dig that @julianolf. How's this?

File: markov.py [hunk 2]
```diff
@@ -116,6 +116,23 @@ def flush_cache(message):
     logger.info('cache cleared')
 
 
+@bot.message_handler(commands=[settings.HELP_COMMAND])
+def help(message):
+    username = bot.get_me().username
```
[the remark this answers is not in the document]
Comment (chrismclennon) [reply markov.py]: eh.

[the remark this answers is not in the document]
Comment (julianolf) [reply markov.py]: I think hound can't see the string interpolation.

**Comment by julianolf** [comment]
Looks great! Thank you @chrismclennon. :)

**Pull request merged**